server/controllers/finance/reports/monthlyBalance/report.handlebars

```diff
@@ -0,0 +1,64 @@
+{{> head title="{{translate 'REPORT.MONTHLY_ACCOUNT_ANALYSIS.TITLE' }}" }}
+
+<div class="container">
+  {{> header}}
+
+  <!-- body -->
+  <div class="row">
+    <div class="col-xs-12">
+      <!-- page title  -->
+      <h3 class="text-center text-uppercase">
+        <strong>{{translate 'REPORT.MONTHLY_ACCOUNT_ANALYSIS.TITLE'}}</strong> <br>
+      </h3>
+      <hr>
+      <h4>
+        <strong>{{this.accountNumber}}: {{this.accountLabel}}</strong>
+      </h4>
+
+      <h5 style="margin:15px; font-weight:bold" class="text-center text-uppercase">
+        {{periodLabel}}
+      </h5>
```

Comment:
Since we select the period before accounts, i think `period` must be above `account`.

Here is a proposal: 
```html
<h3 class="text-center text-bold text-uppercase">{{translate 'REPORT.MONTHLY_ACCOUNT_ANALYSIS.TITLE'}}</h3>
<h3 class="text-center text-uppercase">{{periodLabel}}</h3>
<h4 class="text-center">{{this.accountNumber}}: {{this.accountLabel}}</h4>
```